setup.py

```diff
@@ -27,50 +25,6 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from setuptools import find_packages, setup
-
-tests_require = ['async-solipsism', 'pytest', 'pytest-asyncio', 'pytest-mock']
-docs_require = ['sphinx', 'sphinx-autodoc-typehints', 'sphinxcontrib-asyncio', 'sphinx-rtd-theme']
```

Comment:
sphinx-autodoc-typehints and sphinxcontrib-asyncio are no longer used, so aren't listed in the new version in setup.cfg (they were superseded by native Sphinx support long ago but I forgot to remove them here).